packages/kit/src/vite/index.js

```diff
@@ -388,6 +386,12 @@ function kit() {
 		 * @see https://vitejs.dev/guide/api-plugin.html#configureserver
 		 */
 		async configureServer(vite) {
+			const print_urls = vite.printUrls;
```

Comment:
Should we add an explanatory comment here, why we do this?

```suggestion
			// This method is called by Vite after clearing the screen.
			// This patch ensures we can log any important messages afterwards for the user to see.
			const print_urls = vite.printUrls;
```